
Header header
float64 magX
float64 magY
float64 magZ
